tests/fuzzer.c

```diff
@@ -1960,6 +1960,16 @@ static int basicUnitTests(U32 const seed, double compressibility)
         DISPLAYLEVEL(3, "OK \n");
     }
 
+    {
+        ZSTD_CCtx* cctx = ZSTD_createCCtx();
```

Comment:
minor : 
just FYI,
as a programming pattern, we absolutely love the `const` property, and try to use it anywhere possible.

For example, in this case, the declaration could be : 
`ZSTD_CCtx* const cctx = ZSTD_createCCtx();` .

This means that, after it's initialized, `cctx` address will never change. This is different from the content, which is still mutable.

When used extensively, the `const` property is very useful to protect the code from accidental modifications, and for debugging : now the _absence of_ `const` means that a variable _is __mutable___, and this is a useful signal, as most bugs are concentrated in the mutation of states.
Obviously, this signal is only meaningful if variables are consistently labelled `const` anytime they are.

It's absolutely not a blocker, and I'll validate and merge this patch "as is". This comment is made just to influence your coding style in future contributions.